### DISCUSSION TITLE
Is this new version v20231106 using more vram than the usual using large-v2 or large-v3?

### DISCUSSION THREAD

**Discussion by rrfaria** [openai/whisper#1801] [Q&A]
I have been using the old version with large-v2 on an RTX 3080 TI with 12GB, and it works perfectly. However, now that I updated the version, even using large-v2 doesn't fit in GPU memory. Is there something new in this version that requires more GPU?

![image](https://github.com/openai/whisper/assets/11916455/6082c031-ca58-4b7d-86e1-5f2959df216d)

**Comment by st1782024** [comment]
Hight

**Reply by protektit** [reply]
So let me know Exactly what You are doing and when This occurred.This could be a Huge Bug bounty(I think you can find them a bolt but check back with me and I'll tell you if you can't find). let me just give you the background so somehow they are having problems with their packets but it looks like they're being intentionally switched to the CPU to either ensure that the CPU gets his Traffic it needs to stay viable or that they're doing this to reserve GPU usage for themselves by piggybacking like a small model a llama or something like that(phi?). If you can give me back exactly when they started happening, what model you're using whether you're using one local such as the 8 billion16,0 32 billion parameters and if you're using a gemma three or you're using a hugging face variant. If you're not using a small model, please let me know when this happened and if you can send me screenshots of the usage, which is causing this so like I have, for example if it's a prompt, send me a screenshot of the prompt before you run it and then send me a lifetime video of your metrics and I will work on searching down that bug bounty for you because I think you found it
